### DISCUSSION TITLE
Macros and Pen Pressure Multiple Inputs

### DISCUSSION THREAD

**Discussion by YzooW** [OpenTabletDriver/OpenTabletDriver#1269] [Ideas]
Is it possible to setup macros with OTD?
Also, is it possible to bind multiple keys/commands to different levels of pen pressure like those analog triggers or analog keyboards that have at least two keys/commands depending how far a key/trigger is pressed/pulled?

**Comment by InfinityGhost** [comment]
> Is it possible to setup macros with OTD?

Elaborate on this a bit. I assume you mean with multiple keys for a single binding, which we have implemented in v0.6 (in development)

> Also, is it possible to bind multiple keys/commands to different levels of pen pressure like those analog triggers or analog keyboards that have at least two keys/commands depending how far a key/trigger is pressed/pulled?

This is something we plan on adding but its definitely on the backlog as we have other things that need implementation first. This could however be implemented as a plugin of its own.

**Reply by YzooW** [reply]
Macros similar to those found in gaming mice software. Like a single button can output multiple keys with delays or none in between etc.
